htdocs/mrp/index.php

```diff
@@ -147,6 +148,71 @@
 
 print '<br>';
 
+$sql = "SELECT bom.rowid, bom.ref, bom.label ";
+$sql .= " FROM ".MAIN_DB_PREFIX."bom_bom AS bom, ".MAIN_DB_PREFIX."bom_bomline AS blin, ";
```

Comment:
The table blin, entpt and stk here seems useless. Try to remove this table and their join completely.